set-body/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Should this be here or in the .gitignore root?

Reply:
good point :+1: